universum/modules/automation_server/github_server.py

```diff
@@ -0,0 +1,55 @@
+from ..error_state import HasErrorState
+from ...lib.module_arguments import ModuleArgumentParser
+from .base_server import BaseServerForHostingBuild, BaseServerForTrigger
+
+__all__ = [
+    "GithubServer",
+]
+
+
+class GithubServer(BaseServerForHostingBuild, BaseServerForTrigger, HasErrorState):
+    @staticmethod
+    def define_arguments(argument_parser: ModuleArgumentParser) -> None:
+        parser = argument_parser.get_or_create_group("Github Actions variables",
+                                                     "Github Actions specific parameters")
+
+        parser.add_argument("--github-server", "-ghs", dest="server_url", metavar="GITHUB_SERVER_URL",
+                            help="Github server URL", type=str)
+        parser.add_argument("--github-repo", "-ghr", dest="repo", metavar="GITHUB_REPOSITORY",
```

Comment:
Seems I've seen these arguments somewhere else. Are you sure they are not duplicated?

Reply:
Yes, otherwise we would get a runtime error

Reply:
Do you plan to refactor this in separate PR to remove duplication?

Reply:
No, I don't plan

Reply:
Well, I'd like to leave a note at least somewhere to get back to refactoring it later.

Reply:
Well, I checked the code, and I'm didn't find same argument. Closest to it is: 

```
("--git-repo", "-gr", dest="repo", metavar="GIT_REPO",
 help="See your project home page for exact repository identifier, passed to 'git clone'. "
      "If using SSH, '--git-repo' format is 'ssh://user@server:port/detailed/path'")
```

But it's related to GIT VCS not for CI.  
So I think we don't need to mention this "duplication". Because it's possible to run universum on GA (from one repo) and download  sources from another.

Reply:
Well, then I'd think more about totally different parameters 'git repo' and 'github repo' being called so similarly. I don't have ideas at the moment, but I can clearly see how it can become a problem for the user.

Reply:
What if I just renames parameters to `--github-actions-server|-gas`, `--github-actions-repo|-gar` and etc?

Reply:
In my opinion it should be --github-repo. However, we can add notes to distinguish this and git repo. Note should be added to both parameters.

I prefer shorter form because in the future it can be used for some other case.